eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -18,6 +18,16 @@ jobs:
 
       - template: tools/daily-dev-build-variable/daily-dev-build-variable.yml@azure-sdk-tools
 
+      - script: |
+          pod install
+          xcodebuild -workspace AzureSDK.xcworkspace -scheme ${{parameters.ServiceDirectory}} -destination "platform=iOS Simulator,OS=13.3,name=iPhone 11 Pro Max" -configuration Release -derivedDataPath $(Build.ArtifactStagingDirectory)
```

Comment:
I think you generally need to provide the command at the end of this invocation, e.g. "build" or "archive". I'm guessing that since this works, if you omit it there is an implicit one (probably "build") but we should be explicit.

Reply:
Added.